src/app/shared/components/delete-modal/delete-modal.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, EventEmitter, Output, ViewChild } from '@angular/core';
+import { NgbModal, NgbModalOptions, NgbModalRef } from '@ng-bootstrap/ng-bootstrap';
+import { TestListItem } from '../../interfaces/test-list-item';
+
+@Component({
+  selector: 'app-delete-modal',
+  standalone: true,
+  imports: [],
+  templateUrl: './delete-modal.component.html',
+  styleUrl: './delete-modal.component.css',
+})
+export class DeleteModalComponent {
+  @ViewChild('modal') modal!: NgbModal;
+  protected activeModal?: NgbModalRef;
+  @Output() confirmDeleteEvent = new EventEmitter<boolean>();
+  reports: TestListItem[] = [];
+  deleteQuestion!: string;
+  deleteAllReports!: boolean;
```

Comment:
Also please order these correctly in line with [our conventions](https://github.com/frankframework/frontend-conventions/blob/master/typescript.md#class-member-ordering)

Reply:
Added the change